platform/ios/MGLMapView.mm

```diff
@@ -1897,10 +1897,14 @@ - (void)addAnnotations:(NS_ARRAY_OF(id <MGLAnnotation>) *)annotations
         {
             MGLAnnotationImage *annotationImage = nil;
             NSString *symbolName = nil;
+            NSString *text = nil;
+            if ((MGLPointAnnotation *)annotation != nil) {
+                 text = [(MGLPointAnnotation *)annotation text];
+            }
 
             if (delegateImplementsImageForPoint)
             {
-                annotationImage = [self.delegate mapView:self imageForAnnotation:annotation];
+//                annotationImage = [self.delegate mapView:self imageForAnnotation:annotation];
```

Comment:
This would disable custom annotation images. We’d want to accommodate both options for the annotation API.